### PR TITLE
update document with information about HTTPS certificates

### PR DESCRIPTION
**Pull request by chemaper**
Reviewers

@jesuspg @hmunfru

#### Description
Documentation concerning SSL certificates.

This is the same than the PaaS part, but with some changes in paths and format.

Also fix README.rst link
#### Testing
Tested that the document showed in the browser is OK

### CODE REVIEW

File: doc/installation-guide.rst [hunk 1]
```diff
@@ -37,6 +37,7 @@ It is a a maven application:
        $ mvn assembly:assembly -DskipTests
        
 	   $ cp target/distribution/puppwrapper-dist {folder}
+       $ ./generateselfsigned.sh start on {folder}/puppetwrapper-dist/bin
```
Comment (flopezag) [inline doc/installation-guide.rst]: Where is this script (generateselfsigned.sh)?

Comment (chemaper) [inline doc/installation-guide.rst]: uhm, this is a copy & paste of  the following line, which it's probably copy & paste from puppetwrapper manual.

I'll change the reference to puppetwrapper-dist on these three lines with sdc-server-dist that is the directory actually generated with mvn assembly:assembly

Comment (flopezag) [inline doc/installation-guide.rst]: But you still make reference to puppetwrapper-dist here...

**Comment by flopezag** [comment]
LGTM

**Pull request merged**